scubagoggles/rego/Commoncontrols.rego

```diff
@@ -1574,6 +1988,12 @@ if {
 # GWS.COMMONCONTROLS.12 #
 #########################
 
```

Comment:
The policy API implementation of GWS.COMMONCONTROLS.12.1 does not check for the takeout status of services without individual admin center control (steps 4 and 5 of the [implementation](https://github.com/cisagov/ScubaGoggles/blob/main/baselines/commoncontrols.md#gwscommoncontrols121v03-instructions)). The corresponding setting in the policy API appears to be `takeout_service_status`.

Reply:
done

Reply:
This still isn't right. Say I enable takeout for Blogger, Books, and for services without individual admin control. The details then says, "The following apps with individual admin control have Takeout enabled: Blogger, Google Books." No mention of services without individual admin control. Now say I enable takeout for Blogger and Books, but disable it for services without individual admin control. Now the report goes green, when it should still fail and report that takeout is enabled for Blogger and Books.